recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   skip: true  # [py<38]
-  number: 2
+  number: 3
```

Comment:
If you add `[ci skip] [cf admin skip] ***NO_CI***` to the PR title, that won't trigger a new build to conda-forge. Since you're only updating the tests, a new build probably won't be needed since nothing else in the build step is changing.

Reply:
That's fair, but this package is rather small and I always get nervous about those magic commit commands working, so I tend to just play it safe and bump the build.